warehouse/query-core/src/main/java/datawave/query/jexl/functions/GeoFunctionsDescriptor.java

```diff
@@ -128,7 +144,7 @@ public JexlNode getIndexQuery(ShardQueryConfiguration config, MetadataHelper hel
                                         BoundedRange.create(JexlNodeFactory.createAndNode(Arrays.asList(geLatNode2, leLatNode2)))));
                         
                         // link em up
-                        returnNode = JexlNodeFactory.createAndNode(Arrays.asList(andNode1, andNode2));
+                        returnNode = JexlNodeFactory.createOrNode(Arrays.asList(andNode1, andNode2));
```

Comment:
Maybe rename andNode1 and andNode2 to orNodeN as well?

Reply:
andNode1 and andNode2 are AND nodes though...  lol.  We are ORing together two AND nodes.

Reply:
All the getIndexQuery method is doing is using the parameters from the ASTFunctionNode to generate the node that represents the index query.  It isn't manipulating the tree at all, so flattening isn't really this method's concern.

FunctionIndexQueryExpansionVisitor calls getIndexQuery and ANDs the resulting index query JexlNode with the original ASTFunctionNode.  The original ASTFunctionNode is then replaced by the ANDed ASTFunctionNode+IndexQueryNode combo.  

In short, I don't really see the need to flatten anything here.